CircleciScripts/test_names.json

```diff
@@ -0,0 +1,10 @@
+{
+	"tests" : [
+		{
+			"auth" : "AuthUITest",
+			"api_1" : "AppSyncMutationUITest",
```

Comment:
make api as the category name with the tests in an array.

Reply:
change the type of values of all categories to array:
{
	"tests" : [
		{
			"auth" : ["AuthUITest"],
			"api" : ["AppSyncMutationUITest", "AppSyncQueryUITest"],
			"storage" : ["StorageUITest"]
		}
	]
}